jobs/workflow_cli_release.groovy

```diff
@@ -57,7 +59,11 @@ job("${repoName}-tag-release") {
 
       cat "${GCSKEY}" > ${WORKSPACE}/tmp/key.json
 
-      GIT_TAG="${RELEASE}" make bootstrap build-revision upload-gcs
+      GIT_TAG="${RELEASE}" make bootstrap build-revision fileperms
+
+      ${gcloud} gcloud auth activate-service-account -q --key-file /.config/key.json'
```

Comment:
now that we are referencing groovy vars like `gcloud` (and want interpolation formed), we will need to change the `'''` surrounding this shell step with `"""`, as well as escaping the `$` before any non-groovy (so, bash environment) vars so they don't get interpolated.  (for instance, `"\${RELEASE}"`, etc.)